src/client/common/application/commandManager.ts

```diff
@@ -70,6 +72,7 @@ export class CommandManager implements ICommandManager {
         E extends keyof ICommandNameArgumentTypeMapping,
         U extends ICommandNameArgumentTypeMapping[E]
     >(command: E, ...rest: U): Thenable<T | undefined> {
+        sendTelemetryEvent(Telemetry.CommandExecuted, undefined, { command: command as string });
```

Comment:
I was thinking we'd want this on registerCommand too. That way we'd track command palette calls as well.

Reply:
Or maybe only on registerCommand (otherwise we might have dupes)